docs/data/old-version/dea-fractional-cover-percentiles-landsat-3.0.0/_details.md

```diff
@@ -0,0 +1,143 @@
+## Background
+
+The Fractional Cover (Landsat) product, developed by the Joint Remote Sensing Research Program, provides information about the the proportions of:
+* green vegetation
+* non-green vegetation (including deciduous trees during autumn and dry grass)
+* bare areas
+
+for every 30m x 30m ground footprint.
+
+It provides insight into how areas of dry vegetation and/or bare soil and green vegetation are changing over time.
+
+It is important to be able to analyse and interpret fractional cover. For example, the 90th percentile of bare soil for a particular year will identify areas that have experienced a high portion of bare soil during that year.
+
+## What this product offers
+
+This product is designed to make it easier to analyse and interpret fractional cover. It uses the Fractional Cover (Landsat) product and calculates the statistical summaries (10th, 50th and 90th percentile) of fractional cover per epoch (annual).
+
+This product is available in the following form:
+
+% ## Data description
+
+## Applications
+
+This product provides valuable information for a range of environmental and agricultural applications, including:
+* soil erosion monitoring
+* land surface process modelling
+* land management practices (e.g. crop rotation, stubble management, rangeland management)
+* vegetation studies
+* fuel load estimation
+* ecosystem modelling
+
+## Technical information
+
+### FC-PERCENTILE-ANNUAL-SUMMARY
+
+This contains a (10th, 50th and 90th percentile) of bare, green and non-green vegetation of observations acquired in each full calendar year (1st of January - 31st December) from 1987 to the most recent full calendar year.
+
+This product provides continental composites for each cover fraction.
+
+The percentile ranges for each compositing period can be used to identify the 'greenest' observation i.e. 90th percentile of green cover and 'barest' observation i.e. 90th percentile of bare soil that occurred within the compositing period.
+
+The 10th and 90th percentile are used in preference to the minimum and maximum values because they are less prone to residual noise associated with undetected cloud/cloud shadow. The 50th percentile (median) are used in preference to the mean as the median is less prone to being skewed by extreme values. It is worth noting that some undetected cloud and cloud shadow artefacts may still be present in the 10th and 90th percentiles, especially in areas with frequent cloud cover such as the wet tropics and Tasmania.
+
+To account for satellite availability and status the statistics are calculated using the following satellites for the following periods of time:
+* 1987-1998 : Landsat 5 only
+* 1999 : Landsat 5 and Landsat 7
+* 2000-2002 : Landsat 7 only
+* 2003 : Landsat 5 and Landsat 7
+* 2004-2010 : Landsat 5 only
+* 2011-2012 : Landsat 7 only
+* 2013 onward : Landsat 8 only
```

Comment:
Should Landsat 9 be included in FC percentiles?

Reply:
@tebadi

Reply:
> @tebadi

Landsat 9 doesn't exist in the archived product but I've updated this for the new version.